CHANGELOG.md

```diff
@@ -495,6 +494,8 @@ releases.
       returns temporal info for CMR ISO 19115 SMAP XML files.
     - Updated `@cumulus/cmrjs/cmr-utils.isCmrFilename()` to include
       `isISOFile()`.
+- **CUMULUS-2532**
```

Comment:
How did you figure out this was a part of 9.1? I don't see it on the release notes.https://github.com/nasa/cumulus/releases/tag/v9.1.0

Reply:
Looks like it might be 9.2? https://github.com/nasa/cumulus/blob/master/CHANGELOG.md#v920-2021-06-22

Or do I need new glasses?